### ISSUE TITLE
why 2D variation of the parameters?

### ISSUE THREAD

**Issue by Morisset** [caridlin/Cloudy3D#2]
Hi,
I do not understand why you are running a loop on phi, which does not contain any parameter varying with phi? Why not using a single theta variation, the use the axy-symmetry of the situation to obtain your disk?
Christophe

**Comment by caridlin** [comment]
You are right! That's because I made a mistake in calling the Service_profile_disk function:

Sersic_profile_disk(A, Re, B, n, r_, r_ * np.cos(theta))

which should actually be:

Sersic_profile_disk(A, Re, B, n, np.sqrt((r_ * np.cos(phi)) ** 2 + (r_ * np.sin(phi) / q) ** 2), r_ * np.cos(theta)) where q is the axis ratio!

Thanks and I will update it!

Best,
Zeren
